view_all_inc.php

```diff
@@ -111,10 +111,15 @@
 		<div class="btn-toolbar">
 			<div class="btn-group pull-left">
 		<?php
-			# -- Print and Export links --
+			# -- Print link --
 			print_small_button( 'print_all_bug_page.php', lang_get( 'print_all_bug_page_link' ) );
-			print_small_button( 'csv_export.php', lang_get( 'csv_export' ) );
-			print_small_button( 'excel_xml_export.php', lang_get( 'excel_export' ) );
+			
+			# -- Export links --
+			$t_export_issues_threshold = config_get( 'export_issues_threshold' );
```

Comment:
No need for a variable
`if( access_has_project_level( config_get( 'export_issues_threshold' ) ) ) ...`